packages/stargate/src/stargateclient.ts

```diff
@@ -274,7 +276,7 @@ export class StargateClient {
         }
       : {
           height: response.height,
-          code: response.deliverTx?.code,
+          code: response.deliverTx?.code || 0,
```

Comment:
If we set the default here, could you make the `code` in the interface non-optional?